### PR TITLE
Updated plotting tutorials and images

### PR DESCRIPTION
**Pull request by matteopilz**
I have updated the tutorials involving spectrum plotting. However, I have not added an extra section about the plotting function.

### CODE REVIEW

File: docs/source/datastructures_peak.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ The most important container for raw data and peaks is ``MSSpectrum`` which we
 have already worked with in the `Getting Started <getting_started.html>`_
 tutorial.  ``MSSpectrum`` is a container for 1-dimensional peak data (a
 container of ``Peak1D``). You can access these objects directly, however it is
-faster to use the ``get_peaks()`` and ``set_peaks`` functions which use Python
+faster to use the ``get_peaks()`` and ``set_peaks()`` functions which use Python
 numpy arrays for raw data access. Meta-data is accessible through inheritance
 of the ``SpectrumSettings``  objects which handles meta data of a spectrum.
```
Comment (jpfeuffer) [inline docs/source/datastructures_peak.rst:13]: You might want to use crossreferences to the apidocs:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

File: docs/source/datastructures_peak.rst [hunk 2]
```diff
@@ -18,13 +18,13 @@ displayed.
 
 First we create a spectrum and insert peaks with descending mass-to-charge ratios: 
 
+
 .. code-block:: python
-    :linenos:
```
Comment (jpfeuffer) [inline docs/source/datastructures_peak.rst:22]: why no line numbers?

File: docs/source/datastructures_peak.rst [hunk 3]
```diff
@@ -41,6 +41,31 @@ First we create a spectrum and insert peaks with descending mass-to-charge ratio
...
+    900.0 1.0
+    1000.0 1.0
+    1100.0 1.0
+    1200.0 1.0
+    1300.0 1.0
+    1400.0 1.0
+    700.0 1.0
+
+
 Note how lines 11-12 (as well as line 19) use the direct access to the
```
[the remark this answers is not in the document]
Comment (matteopilz) [reply docs/source/datastructures_peak.rst:69]: I want to include the line numbers, but the preview didn't work with them in it: https://github.com/matteopilz/pyopenms-docs/blob/master/docs/source/datastructures_peak.rst

File: docs/source/datastructures_peak.rst (hunk 1 above)
Comment (matteopilz) [reply docs/source/datastructures_peak.rst]: https://github.com/OpenMS/pyopenms-docs/blob/master/docs/source/datastructures_peak.rst

Comment (matteopilz) [reply docs/source/datastructures_peak.rst]: I tried to set up a readthedocs to test this out, but it doesn't take the most recent version, beside build triggering on every push. https://mp-pyopenms-docs.readthedocs.io/en/latest/datastructures_peak.html

File: docs/source/datastructures_peak.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (jpfeuffer) [reply docs/source/datastructures_peak.rst:69]: GitHub does not have full RST support. There will always be issues. The docs are not made to be viewed in GitHub.
I think you will have a much better debugging experience if you build the project and have look at the generated HTML pages:
https://github.com/OpenMS/pyopenms-docs/blob/master/docs/README.md

[the remark this answers is not in the document]
Comment (jpfeuffer) [reply docs/source/datastructures_peak.rst:69]: Why not work on GitPod? gitpod.io#https://github.com/OpenMS/pyopenms-docs/pull/325

File: docs/source/datastructures_peak.rst (hunk 1 above)
Comment (matteopilz) [reply docs/source/datastructures_peak.rst]: I tried writing `.. py:function:: get_peaks()`, but it didn't work.

File: docs/source/datastructures_peak.rst [hunk 4]
```diff
@@ -53,6 +49,28 @@ First we create a spectrum and insert peaks with descending mass-to-charge ratio
...
+
+    First peak: 500.0 1.0
+
+
+Note how lines 11-12 (as well as line 19) use the direct access to the
+``Peak1D`` objects (explicit iteration through the ``MSSpectrum`` object, which
+is convenient but slow since a new ``Peak1D`` object needs to be created each
+time).
+The following example uses the faster access through numpy arrays with .. py:function:: get_peaks() or
+.. py:function:: set_peaks(). Direct iteration is only shown for demonstration purposes and should not be used in
```
Comment (jpfeuffer) [inline docs/source/datastructures_peak.rst:61]:
```suggestion
The following example uses the faster access through numpy arrays with :py:meth:`~.MSSpectrum.get_peaks` or
:py:meth:`~.MSSpectrum.set_peaks`. Direct iteration is only shown for demonstration purposes and should not be used in
```

Per the documentation that I linked:
- .. py.*** is for definitions
- :py:*** is for references
- func is for functions. Those are methods -> meth
- use "~" prefix to not spell the fully qualified name

**Review by jpfeuffer — CHANGES_REQUESTED** [review]
You do not need to set up an own Readthedocs server. You just need to build the html documentation and then open with your browser. As per docs:

```bash
git clone https://github.com/OpenMS/pyopenms-docs && cd pyopenms-docs/docs
pip install -r requirements.txt
make html
open build/html/index.html
```

**Comment by matteopilz** [comment]
I have added the cross-references. Currently, `MSExperiment` and the `help` function (although we probably don't need that one) from Python are not working.

**Comment by jpfeuffer** [comment]
Cool. Yes we can have a look on a few non-working references later.

**Comment by matteopilz** [comment]
So I guess we can merge that. I would also add cross-references to and update the other pages, but maybe in a new PR.

**Review by jpfeuffer — APPROVED** [review]

**Pull request merged**